# Lab | Inferential statistics


### Instructions

1. It is assumed that the mean systolic blood pressure is `μ = 120 mm Hg`. In the Honolulu Heart Study, a sample of `n = 100` people had an average systolic blood pressure of `130.1 mm Hg` with a standard deviation of `21.21 mm Hg`. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

   - Set up the hypothesis test.
   - Write down all the steps followed for setting up the test.
   - Calculate the test statistic by hand and also code it in Python. It should be 4.76190. We will take a look at how to make decisions based on this calculated value.

The hypothesis that we would like to disprove is that the systolic blood pressure of the people in the Honolulu Heart Study is significantly different from the population systolic blood pressure, so:

* H0 is $\mu$(sample) != $\mu$(population)
- H1 is $\mu$(sample) = $\mu$(population)

Given this is a medical case, we'll look for a statistical significance >99%, so a p-value <1%.

In [2]:
import math

pop_mean = 120
sample_mean = 130.1
sample_std = 21.21
n = 100

t = abs(pop_mean - sample_mean) / (sample_std / math.sqrt(n))
print(t)

4.761904761904759


For a two-tailed test, a t value of 4.76 corresponds to a p-value < 0.1%, which means that we can reject the null hypothesis.